content/tokio/topics/bridging.md

```diff
@@ -21,13 +21,16 @@ project.
 The `#[tokio::main]` macro is a macro that replaces your main function with a
 non-async main function that starts a runtime and then calls your code. For
 instance, this:
+
 ```rust
 #[tokio::main]
 async fn main() {
     println!("Hello world");
 }
 ```
+
 is turned into this:
+
 ```rust
```

Comment:
I'm not a fan of this lint. There's nothing wrong with not having an empty line before/after a code block.

Reply:
> nothing wrong with not having an empty line

CommonMark spec [agrees with this (4.5)](https://spec.commonmark.org/0.30/)! Just the markdownlint extension has an opinion on this which I took for granted: https://github.com/DavidAnson/markdownlint/blob/main/doc/md031.md

Reply:
> nothing wrong with not having an empty line

CommonMark spec [agrees with this (4.5)](https://spec.commonmark.org/0.30/)! Just the markdownlint extension has an opinion on this which I took for granted: https://github.com/DavidAnson/markdownlint/blob/main/doc/md031.md